.travis.yml

```diff
@@ -16,6 +16,13 @@ matrix:
       services: docker
       env: PYTHON=python
 
+    # Linux Python 3
+    - sudo: required
+      language: python
+      python: 3.5
+      services: docker
+      env: PYTHON=python
+
```

Comment:
Why is this added?

Reply:
because `run` function is introduced to python in version 3.5 and I would like to run new test. For python 2.7 and 3.4 I should work with Popen object which consume much more time and create hard to read code. And If I good understand, the support of python 3.4 will be dropped in short time.

Reply:
OK, I understand now. Didn't see that `subprocess.run`. But we still want to support 2.7 for a while, I believe. So let's bump 3.4 to 3.5 for CI in the PR that's dropping 3.4, no?

Can't you just use `check_output(..., stderr=subprocess.STDOUT)` as suggested on https://docs.python.org/3.4/library/subprocess.html#subprocess.check_output?

Reply:
But in this way I cannot test if message is in proper pipe. I also not sure how it works with mixing pipes. It is rare, but possible (experience from C, maybe python handle it better) to have inclusion `first_part something_from_stdin second_part` instead of `first_part second_part`

Reply:
That should be OK, no? Python is better at these kinds of things and there's no multi-threading or so going on, so nothing's writing at the same time to the stdout/stderr. Also, you know it's in the `stderr` because you're writing there, no?
Anyway, we still need to support 2.7?

Btw, even in Python 2.7 and 3.4, you already have `universal_newlines=True` to get a nice string back, rather than bytes.

Reply:
I will check this option tomorrow. 

There are many software in python 2. So i Think that in moment when cibuildwheel drop python 2.7 support, then last version supporting python 2.7 should be refered in documentation for at least one year.

Reply:
@YannickJadoul  Thanx for `universal_newlines` parameter.

Reply:
Great! So can we then remove this extra configuration block for Travis CI, here, if the code now also works with Python 3.4?

Reply:
@YannickJadoul 
No. `universal_newlines` help to remove decoding bytes. There is no `run` function in python 3.4. Python 3.4 reach end of life. Of course it is possible to write code compatible with python 3.4 but if it is needed? Other approach is to not test this utils on travis.

Reply:
Aha, and you don't want to try `check_output(..., stderr=subprocess.STDOUT)`?

If not, I suggest still keeping only one Travis CI configuration, that you change to 3.5, and then then maybe we'll let the depend on #168 being merged.

Or maybe @joerick has a better idea/different judgement.

Reply:
Problem with `stderr=subprocess.STDOUT` is that `stdout` is not printed immediately but buffered (for performance reason). So even than there is no parallelism, then the `stdout` and `stderr` can print mixed. 

Also I think that it is important to print all such communicates to `stderr`, and `check_output(..., stderr=subprocess.STDOUT)` do not check if it is done properly.

Reply:
Ok, yes, I hadn't thought about that.

Sorry for being so annoying about this detail. It just feels like a bit of a waste to run all tests multiple times... :)

Reply:
Happy to remove 3.4 from project CI. Support will be removed soon anyway

Reply:
Removed